connect-app/FirebaseConfig.js

```diff
@@ -0,0 +1,19 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getFirestore } from "firebase/firestore";
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
```

Comment:
Is this all firebase provided manual Config during setup?

Reply:
Yes, this config is from Firebase documentation to integrate it in our app

Reply:
okay